package.json

```diff
@@ -121,6 +122,20 @@
     },
     "test:browser-ci": {
       "command": "karma start",
+      "dependencies": [
+        "clean-test-failures-file",
```

Comment:
always delete failure tests on the first run, subsequent runs on failures will be done by running the `test:browser-ci:retry` script.